CentralComputing/tests/UtilsTest.cpp

```diff
@@ -80,8 +80,12 @@ TEST(UtilsTest, busyWaitIndivThreads) {
 
 	EXPECT_EQ(sched_setaffinity(0, sizeof(first), &first), 0);
 }
-TEST(UtilsTest, GPIOToggleTest) { //Test should work on BBB but not necessarily local systems
-	const int HEARTBEAT_GPIO = 37;
+
+
+//Test should work on BBB. Otherwise returns true
+TEST(UtilsTest, GPIOToggleTest) {   
+
+  #ifdef BBB
```

Comment:
This is how you differentiate between on the BBB, and not on the BBB.

I just added this `#define BBB` when the BBB compiler is used, so this should work